_posts/2022-02-19-newpipe-0.22.0.md

```diff
@@ -0,0 +1,155 @@
+
+### Fixed
+
+- As the first entry in our big series of fixes this release, [@litetex](https://github.com/litetex) disabled an unreliable test for the SoundCloud stream extractor. Who even needs tests? (Just kidding, they're a cornerstone of good programming. ~~Eat your broccoli~~ Write your unit tests, kids.)
+
+- When a video link was opened with a timestamp URL parameter that had more than 3 digits, NewPipe ignored the 4th and higher digits. This meant that the app took you to the wrong timestamp. For example, 1001 seconds (16 minutes and 41 seconds) would be parsed as 1 second instead. [@mhmdanas](https://github.com/mhmdanas) has fixed this.
+
+- A nasty bug a lot of our users faced was that switching from one player type to another caused the playback to restart from the beginning. This has been fixed by [@litetex](https://github.com/litetex).
+
+- [@Stypox](https://github.com/Stypox) fixed the order of local search results from your search history, so now they are back to being ordered by creation date instead of alphabetical order (which made it seem like new entries weren't being added to the search result history database, unless you searched for a video titled "AAAA" or something).
+
+ -[@XiangRongLin](https://github.com/XiangRongLin) ensured that search history entries in tests are in the correct order, fixing the sometimes-failing-sometimes-passing CI pipeline.
+
+- [@Stypox](https://github.com/Stypox) fixed many different crashes where the code expected the player's queue to be not `null` but it was, indeed, `null`.
+
+- [@Cheechaii](https://github.com/Cheechaii) introduced #respect into the app. Now the player respects display cutouts (like notches) when playing in multi-window mode, whereas previously, black bars were shown.
+
+- Speaking of hashtags, [@Jaspann](https://github.com/Jaspann) made their first contribution to a public open source project (Congratulations!): They fixed hashtags that use non-English characters. Thank you!
+
+- [@Jaspann](https://github.com/Jaspann), in their second contribution, fixed showing the PeerTube-specific sub-channel icon on non-PeerTube services, where it makes no sense.
+
+- [@TobiGr](https://github.com/TobiGr) fixed a link in the English README linking to the Portuguese README. Not really their first/second contribution to the project, a welcome one nonetheless
```

Comment:
Really unimportant, I'd move it down as much as possible, maybe into the localisation section

Reply:
So now you destroyed that joke opportunity pfff.